packages/builders/__tests__/components/selectMenu.test.ts

```diff
@@ -55,17 +55,26 @@ describe('Button Components', () => {
 				description: 'test',
 			};
 
-			const selectMenuData: APISelectMenuComponent = {
+			const selectMenuDataWithoutOptions = {
 				type: ComponentType.SelectMenu,
 				custom_id: 'test',
 				max_values: 10,
 				min_values: 3,
 				disabled: true,
-				options: [selectMenuOptionData],
 				placeholder: 'test',
+			} as const;
+
+			const selectMenuData: APISelectMenuComponent = {
+				...selectMenuDataWithoutOptions,
+				options: [selectMenuOptionData],
 			};
 
-			expect(new SelectMenuComponent(selectMenuData).toJSON()).toEqual(selectMenuData);
+			expect(
+				// @ts-expect-error
```

Comment:
Whats the reason for this? is it just because selectWithoutOptions doesn't have options defined?